cf-ops-automation-cassandra-broker/src/test/java/com/orange/oss/cloudfoundry/broker/opsautomation/ondemandbroker/sample/CassandraServiceProvisionningTest.java

```diff
@@ -164,14 +164,20 @@ public void initPaasTemplate(Git git) {
             git.checkout().setName("develop").setCreateBranch(true).call();
 
             //root deployment
-            Path coabDepls = gitWorkDir.toPath().resolve(CassandraProcessorConstants.ROOT_DEPLOYMENT_DIRECTORY);
+            Path coabDepls = gitWorkDir.toPath().resolve(deploymentProperties.getRootDeployment());
             //sub deployments
             Path templateDir = coabDepls
-                    .resolve(CassandraProcessorConstants.MODEL_DEPLOYMENT_DIRECTORY)
-                    .resolve(CassandraProcessorConstants.TEMPLATE_DIRECTORY);
+                    .resolve(deploymentProperties.getModelDeployment())
+                    .resolve(deploymentProperties.getTemplate());
             createDir(templateDir);
-            createDummyFile(templateDir.resolve(CassandraProcessorConstants.MODEL_MANIFEST_FILENAME));
-            createDummyFile(templateDir.resolve(CassandraProcessorConstants.MODEL_VARS_FILENAME));
+            createDummyFile(templateDir.resolve(deploymentProperties.getModelDeployment() + DeploymentConstants.YML_EXTENSION));
+            createDummyFile(templateDir.resolve(deploymentProperties.getModelDeployment() + DeploymentConstants.HYPHEN + deploymentProperties.getVars()+ DeploymentConstants.YML_EXTENSION));
+
+            Path operatorsDir = coabDepls
```

Comment:
@JCL38-ORANGE would be good to extract this part into the Secrets/Template Generator or its tests